modules/platforms/cpp/ignite/common/mpi.h

```diff
@@ -0,0 +1,222 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <mbedtls/bignum.h>
+
+#include <cstddef>
+#include <cstdint>
+#include <string>
+#include <utility>
+
+namespace ignite {
+
+/**
+ * Enum with the possible values of sign.
+ */
+enum mpi_sign : short { POSITIVE = 1, NEGATIVE = -1 };
```

Comment:
Why is it `short`?

Reply:
Just because sign field in the MbedTLS mpi is short. To avoid conversion warnings.